main.tex

```diff
@@ -108,13 +131,17 @@ \section{Introduction}\label{sec:introduction}
 \begin{enumerate}
  \item Zero determinant strategies are not particularly effective for $N > 2$
  \item Complex strategies can be effective, and in fact can naturally evolve
-     through evolutionary processes to outperform intelligently designed
-     strategies.
- \item Strong resistors specifically evolve or have a handshake mechanism. This
-     has the potential to offer insight in to organisms such as antibacterial
-     resistant bacteria \cite{Davies2010}.
+     through evolutionary processes to outperform designed strategies.
+ \item Strong resistors specifically evolve or have a handshake mechanism.
+ \item Strong invaders are generally cooperative strategies that do not defect
+ first but retaliate to varying degrees of intensity against strategies that defect.
+ \item Strategies evolved to maximize their total payout can be strong invaders
+ and achieve mutual cooperation with many other strategies.
 \end{enumerate}
 
+These finding have the potential to offer insight in to organisms such as antibacterial
```

Comment:
My understanding of antibacterial resistant bacteria, is that they are now a huge problem because of the fact that they have evolved to resist antibiotics: they are in effect organisms that have evolved to be strong at resisting invasion.

I saw a strong parallel with what we're observing here with our strategies: and what I'm postulating is that these resistant bacteria would have some sort of handshaking mechanism (similar to our strategies). 

It's a bit of a step but it's what the evolution of our handshakes made me think about (the resistant bacteria have evolved "something" to become a big problem).